Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -176,22 +186,27 @@ public struct Profile: Equatable {
     }
 }
 
-extension Event.EventName {
-    var value: String {
+extension Event.Legacy.MetricName {
+    public var value: String {
         switch self {
         case .OpenedPush: return "$opened_push"
         case .ViewedProduct: return "$viewed_product"
-        case .SearchedProducts: return "$searched_products"
         case .StartedCheckout: return "$started_checkout"
-        case .PlacedOrder: return "$placed_order"
-        case .OrderedProduct: return "$ordered_product"
-        case .CancelledOrder: return "$cancelled_order"
-        case .PaidForOrder: return "$paid_for_order"
-        case .SubscribedToBackInStock: return "$subscribed_to_back_in_stock"
-        case .SubscribedToComingSoon: return "$subscribed_to_coming_soon"
-        case .SubscribedToList: return "$subscribed_to_list"
-        case .SuccessfulPayment: return "$successful_payment"
-        case .FailedPayment: return "$failed_payment"
+        case .AddedToCart: return "$added_to_cart"
+        case .ActiveOnSite: return "$active_on_site"
+        case let .CustomEvent(value): return "\(value)"
+        }
+    }
+}
+
+extension Event.V1.MetricName {
+    public var value: String {
+        switch self {
+        case .OpenedPush: return "$opened_push"
+        case .ViewedProduct: return "Viewed Product"
+        case .StartedCheckout: return "Started Checkout"
+        case .AddedToCart: return "Added to Cart"
+        case .ActiveOnSite: return "Active on Site"
```

Comment:
"Opened App"

Reply:
Following up on Evan's comment, based on our chat in standup today, I think we should switch "Active on Site" to "Opened App." That follows Klaviyo's standard naming convention of Verb Noun (e.g., Received Push) and feels generic enough to apply to other potential use cases beyond mobile apps in the future.